csfieldguide/static/interactives/picture-memory/README.md

```diff
@@ -0,0 +1,5 @@
+# Picture Memory Interactive
+
+**Author:** Courtney Bracefield
+
+This interactive demonstrates the limitations of short term memory by getting the user to remember a number of items within 30 seconds (without writing anything down).
```

Comment:
Maybe mention where the assets are from? It's in the page footer but associating them with the interactive as well would be good